RentalCRUD.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Newtonsoft.Json;
+using ProyectoAgencia.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+
+
+namespace ProyectoAgencia
+{
+    public class RentalCRUD :IRentalCRUD
+    {
+        List<Rental> allRentals = new List<Rental>();
+        private string rentalJsonFile = ConfigurationManager.AppSettings[("pathRentals")];
+        string rentalJsonString = File.ReadAllText("rental.json");//I didn't use an "using" block because File.ReadAllText close the file at the end.
+        Rental rental = new Rental();
+
+
+        public Rental Get(string Id)
```

Comment:
Por qué no utilizar `int` como tipo de dato para el campo `Id` ?
```suggestion
        public Rental Get(int Id)
```